homeassistant/components/reolink/entity.py

```diff
@@ -128,3 +128,13 @@ def __init__(
                 sw_version=self._host.api.camera_sw_version(dev_ch),
                 configuration_url=self._conf_url,
             )
+
+    async def async_added_to_hass(self) -> None:
+        """Entity created."""
+        await super().async_added_to_hass()
+        cmd_key = self.entity_description.cmd_key
+        if (
+            cmd_key is not None
+            and self._channel not in self._host.update_cmd_list[cmd_key]
+        ):
+            self._host.update_cmd_list[cmd_key].append(self._channel)
```

Comment:
Does this need to be cleaned up when the entity is removed from hass?

Reply:
I have been thinking about this since it would be ideal, but that is not possible.
There are multiple entities using the same update_cmd, for instance the AI sensitivity for people, vehicle, pet, animal and package are all updated using a single command.
When adding the entities the command is only added 1x (per channel) to the list.
Upon removing I don't know if there are still other entities using that command or that this was the last one which removed itself.

After a integration restart or HomeAssistant restart the commands will not be send anymore when not needed....

Reply:
Or I would have to keep a second list which counts how many entities are beeing added per command, if the counter hits 0 I can remove it from the update_cmd_list, I will work on this.
But maybe keep that extra logic for a seperate PR, or do you want to include it in this one?